gcp_variant_transforms/libs/preprocess_reporter.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Generates preprocess report.
+"""Generates preprocess report (sorted).
```

Comment:
nit: the 'sorted' comment is strange here as it can confuse the user reading this about what aspects are sorted. I suggest removing it or (if you really want :p ) explaining what parts are sorted.